docs/query-builder.txt

```diff
@@ -81,6 +86,33 @@ of the Quick Start.
 To perform read and write operations by using the query builder, import the
 ``Illuminate\Support\Facades\DB`` facade and compose your query.
 
+.. _laravel-options-query-builder:
+
+Set Query-Level Options
```

Comment:
S: I think it makes more sense to have the "Retrieve Matching Documents" section come before this one, since "Set Query-Level Options" sets options for where(), described in "Retrieve Matching Documents". Maybe you could move this section before "Modify Query Results"?